j2k/src/org/jetbrains/kotlin/j2k/ExpressionConverter.kt

```diff
@@ -501,6 +501,18 @@ class DefaultExpressionConverter : JavaElementVisitor(), ExpressionConverter {
                 }
             }
 
+            if (type?.canonicalText == CommonClassNames.JAVA_LANG_STRING) {
+                val argument = expression.argumentList?.expressions?.singleOrNull()
+                if (argument != null && argument.type?.canonicalText == CommonClassNames.JAVA_LANG_STRING) {
+                    result = if (argument is PsiLiteralExpression) {
+                        codeConverter.convertExpression(argument)
+                    } else {
+                        LiteralExpression("${CommonClassNames.JAVA_LANG_STRING}(${argument.text}) as String").assignPrototype(expression)
```

Comment:
Isn't it better to create valid AST here? Via `MethodCallExpression`, `TypeCastExpression` and so on?